From fd38c58c0cffe0d2d24f27229c978fe959285c33 Mon Sep 17 00:00:00 2001
From: jbettini <ashhxtm@gmail.com>
Date: Fri, 30 Aug 2024 06:30:53 +0200
Subject: [PATCH] Add patch to modify EXTRAVERSION in Makefile for kernel
 version string

This commit updates the EXTRAVERSION field in the Makefile to append "-thor_kernel" to the kernel version string.

Signed-off-by: jbettini <ashhxtm@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7b60eb103..07a222d9b 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 11
 SUBLEVEL = 0
-EXTRAVERSION = -rc5
+EXTRAVERSION = -rc5-thor_kernel
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.44.0

